.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: php
+php:
+  - '5.4'
+  - '5.5'
+  - '5.6'
+  - '7.0'
+script: 
+- phpunit Tests
```

Comment:
Can you add a new line at the end of file, it's best to have a trailing new line as a standard in all files.